packages/styleguide/src/examples/pages/aui-calendar/aui-calendar.page.ts

```diff
@@ -1,12 +1,20 @@
 import {Component} from '@angular/core';
 import {DateRange} from '@acpaas-ui/js-date-utils';
+import {IntervalBuilder} from '@acpaas-ui/ngx-utils';
+import * as _moment from 'moment';
```

Comment:
Mag ook gewijzigd worden

Reply:
dit is een example pagina & for convenience sake wordt moment gebruikt.